src/elements/ComicScroll.tsx

```diff
@@ -1,19 +1,19 @@
 import ComicCard from "../atoms/ComicCard";
-import { IComicData } from "../models/comicData";
+import { ICarouselComic } from "../models/homePageData";
 function ComicScroll({
   heading,
   comics,
 }: {
   heading: string;
-  comics: IComicData[];
+  comics: ICarouselComic[];
 }) {
   return (
     <>
       <h4 className="text-medium font-medium">{heading}</h4>
       <div className="major-cities-scroll bg-white overflow-auto whitespace-nowrap mb-6">
```

Comment:
I am a bit confused about the class name ` major-cities-scroll `. Are the styles defined in your css files or in the tailwind config files?

Reply:
Oh i forgot to remove it when I copied it from my weather app. But I'll deal with UI stuff in another PR